src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java

```diff
@@ -4,25 +4,25 @@
 import static seedu.taassist.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Tag in TA-Assist.
+ * Represents a Class in TA-Assist.
  * Guarantees: immutable; name is valid as declared in {@link #isValidModuleClassName(String)}
  */
 public class ModuleClass {
 
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
+    public static final String MESSAGE_CONSTRAINTS = "Class names should be alphanumeric";
     public static final String VALIDATION_REGEX = "\\p{Alnum}+";
 
-    public final String tagName;
+    public final String className;
 
     /**
      * Constructs a {@code Tag}.
      *
-     * @param tagName A valid tag name.
+     * @param className A valid tag name.
```

Comment:
Maybe need to change to 'A valid class name.' too?

Reply:
Yes, thanks for pointing out!